docs/UserGuide.md

```diff
@@ -350,15 +382,17 @@ Finds persons who contain any of the given keywords.
 - Only full words will be matched e.g. `Han` will not match `Hans`
 - Persons matching at least one keyword will be returned (i.e. `OR` search).
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+- Comment is also searched. e.g. `peanuts` will match `Allergic to peanuts`
 
 **Aliases:** `f`
 
 **Examples:**
 
-- `find John` returns `john` and `John Doe`
-- `find alex david` returns `Alex Yeoh`, `David Li`<br>
+- `find John` includes `john` and `John Doe`
+- `find alex david` includes `Alex Yeoh`, `David Li`<br>
   ![img.png](images/findAlexDavid.png)
-- `find participant` returns all participants in the list
+- `find participant` includes all participants in the list
+- `find 1` includes the person with group number 1
```

Comment:
"includes" does not seem to be the right verb here. I suggest we keep the same verb as AB-3.

Reply:
It was intentional as we support other information(email, comment,...) that was not supported in AB-3. the word participant might be included inside comment, so return is too restrictive word.

Reply:
I will change to "returns a list that includes"

Reply:
The name here refers to the whole contact with all of their information, not only name. I think your added examples also explain that.